.github/ISSUE_TEMPLATE/tag-suggestion.yml

```
@@ -0,0 +1,17 @@
+name: Tag Suggestion
+description: Suggest a new tag.
+title: I DID NOT READ THE INSTRUCTIONS
```

I would suggest making it clearer, as this currently sounds like an "inside joke". (The fact that people never read the instructions, so we made this)

For example
```suggestion
title: TAG ISSUES GO TO THE META REPO, PLEASE READ THE FOLLOWING INSTRUCTIONS
```